### ISSUE TITLE
Conditional imports are not recognized

### ISSUE THREAD

**Issue by redDwarf03** [Dart-Flutter-Pub-dev/daunused#1]
Hello

thanks for this lib!
Just, when we have a conditional import, the dart are not recognized
Example: 
```
import 'package:aedex/util/browser_util_desktop.dart'
    if (dart.library.js) 'package:aedex/util/browser_util_web.dart';
```
```browser_util_desktop.dart``` and ```browser_util_web``` are in the list of unused files or it's not the case

Thx

**Comment by mauriciotogneri** [comment]
@redDwarf03 Thanks for submitting the bug. I'll look into it.

**Comment by mauriciotogneri** [comment]
@redDwarf03 I released version `1.6.1`. Let me know if the problem persists.

By the way, here you can find other packages that I have done, in case you find one useful:
https://pub.dev/publishers/mauriciotogneri.com/packages
